setup.cfg

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2019 The curiu authors
+#
+# SPDX-License-Identifier: LicenseRef-ISC-curiu
+
+[metadata]
+name = cuiru
+license = ISC
+license_file = LICENSE.txt
+description = Experimental code for the conda package mananger.
+long_description = file: README.md, AUTHORS.md, CHANGELOG.md
+long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+url = https://github.com/curiu/curiu
+author = Marcel Bargull
+author_email = marcel.bargull@udo.edu
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: ISC License (ISCL)
+    Natural Language :: English
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: System :: Software Distribution
+
+[options]
+install_requires =
+    conda ==4.7.12
+python_requires = >=3.7
+setup_requires =
+    setuptools >=38.6.0  # support for long_description_content_type
+    # cffi >=1.0.0
+# include_package_data = True
+packages = find:
+package_dir =
+    = src
+
+[options.packages.find]
+where = src
+
+[options.entry_points]
+console_scripts =
+    curiu = curiu.ui.cli:main
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = src/curiu/_version.py
+versionfile_build = curiu/_version.py
+tag_prefix = v
+parentdir_prefix = curiu-
+
+[flake8]
+max-line-length=96
```

Comment:
I am pretty sure that we need to remove some warnings here because ``black`` violate some warnings of ``flake8``

Reply:
we need to add
```
ignore = E501,W503,E203
```

Reply:
`E501` should be fine due to the `max-line-length` config.
But we possibly need others too, I assume. Let's just add them whenever they come up.

Reply:
Sure thing! :)